force-app/main/default/classes/DMLOperations.cls

```diff
@@ -47,9 +50,13 @@ public with sharing class DMLOperations {
      * @param industry The industry of the account to be created.
      */
     public static void createAccount(String name, String industry) {
-        // Create a new Account and set the fields inline
 
+        Account newAccount1 = new Account();
+        // Create a new Account and set the fields inline
+        newAccount1.Name = name;
+        newAccount1.Industry = industry;
         // Insert the Account
```

Comment:
Is there another way that you set these values right in the Account constructor? (inline)